spec/classes/mountpoints/tmp_spec.rb

```diff
@@ -80,18 +87,12 @@
               })
             end
 
-            it {
-              is_expected.to contain_systemd__unit_file('tmp.mount').
-                with_content(
-                  <<-EOM
-[Mount]
-What=tmpfs
-Where=/tmp
-Type=tmpfs
-Options=mode=1777,nodev,noexec,nosuid
-                  EOM
-                )
-            }
+            it { is_expected.to contain_mount('/tmp').with({
+              :options => 'data=ordered,nodev,noexec,nosuid,relatime,rw,seclabel',
+              :device  => '/dev/sda3'
```

Comment:
Do we not need to be checking `mode` as well?

Reply:
@sharkbruhaha Not really. That isn't defined by variable logic and we would just then be re-checking something that's "just puppet".

This gets back to the maintainability of 100% code coverage.